src/rag_processor/core/sentry.py

```
@@ -148,17 +152,22 @@ def _get_release_version() -> str:
     try:
         import subprocess
 
-        sha = subprocess.check_output(
-            ["git", "rev-parse", "--short", "HEAD"],
-            stderr=subprocess.DEVNULL,
-        ).decode().strip()
+        sha = (
+            subprocess.check_output(
+                ["git", "rev-parse", "--short", "HEAD"],
```

Ruff (0.14.8)

Starting a process with a partial executable path

(S607)